lib/jiraConfig.js

```diff
@@ -14,7 +14,7 @@
                 this.authorization = 'Basic ' + btoa( options.authorization.username + ':' + options.authorization.password);
             }
             this.ssl = this.resturi.match(/^https/) !== null ? true : false;
-            var path = "/jira/rest";
+            var path = "/rest";
```

Comment:
Default is `/jira/rest`, if needed we can configure it through arguments for path configuration.